driver/config/config_test.go

```diff
@@ -946,7 +947,8 @@ func TestIdentitySchemaValidation(t *testing.T) {
 		assert.NoError(t,
 			os.MkdirAll(tdir, // DO NOT CHANGE THIS: https://github.com/fsnotify/fsnotify/issues/340
 				os.ModePerm))
-		tmpConfig, err := os.Create(filepath.Join(tdir, "config.yaml"))
+		configFileName := randx.MustString(8, randx.Alpha)
```

Comment:
This test https://github.com/ory/kratos/actions/runs/3605104374/jobs/6075180290#step:15:1429 has been flaky a few times. I couldn't repro it locally, but suspect that using just `config.yaml` in the same directory might lead to flakyness. This should solve this. Other opinions welcome! :)